docs/how-to/publishing/android.rst

```diff
@@ -0,0 +1,132 @@
+=======
+
+Overview
+--------
+
+The Google Play Store is the most widely-used Android app store. This guide
+focuses on how to distribute a BeeWare app there.
+
+Build the app in release mode
+-----------------------------
+
+Run this command::
+
+  briefcase publish android
+
+This will result in four APK files being generated. Each file corresponds
+to a different type of CPU that may be in an Android device, known as an
+Android ABI.
+
+.. TODO list:
+    - Adjust cookiecutter to make per-ABI APKs, per https://dev.to/sys1yagi/enable-split-apk-only-when-release-build-1ij
+    - Use Windows/Mac/Linux tabs for the above.
+    - Test it.
+
+
+Sign the APK files
+------------------
+
+.. admonition:: Create code signing identity
+
+  Before you sign the APK files, you need to :doc:`create a code signing
+  identity<../code-signing/android>`.
+
+Since you already have a code signing identity, we run the two Android SDK
+commands required to sign the APK files. First, we ensure the APK files are
+aligned to 32-bit boundaries, per the Android requirements.
+
+::
+
+  $ zipalign -p 4 -f filename.apk
+
+Next, we sign the actual package files.
+
+::
+
+  $ apksigner sign --ks my-release-key.jks my-app-unsigned-aligned.apk
+
+.. TODO list:
+    - Use Windows/Mac/Linux tabs for the above.
+    - Make sure we specify a path that finds these programs.
+    - Test it.
+    - Decide with Russell where we want to store these keys.
+
+Adding the app to the Google Play store
+---------------------------------------
+
+Adding the app requires a few steps. You need to have a Google Account and
+create a listing for your app. You will need to choose your app's signing
+configuration in the store. Finally, you will create a new release of the app,
+at which point you will upload the APK files that comprise your app.
+
+Creating a listing
+++++++++++++++++++
+
+To distribute your app on the Google Play Store, you need to visit
+https://play.google.com/apps/publish/ and create an account.
+
+Once you've done that, click **Create Application**. Choose a language and
+write a brief app title, up to 50 characters. This should probably be the
+same as your app's Formal Name in the BeeWare ecosystem.
+
+This will take you to **Store Listing** section of your app. You will need to
+answer a variety of questions. Take your time to answer them well.
+
+You will need to upload an icon. For briefcase apps, an icon is typically specified in
+your app's ``pyproject.toml`` file. Look for a line starting with ``icon = ``.
+The Google Play Store will require a variety of other information, including at least
+one screenshot of your app and your assessment of likelihood that children will want
+to use the app.
+
+Configure app signing
++++++++++++++++++++++
+
+Click **App Signing** in the left navigation. The Google Play Store offers an app signing
+feature where they maintain the signing key for your app. For simplicity, you will
+click **Opt Out** to opt out of that feature.
```

Comment:
This appears out of order; I couldn't manage keys until uploading an APK.

Reply:
Fixed now, I believe.